rpc-rs/src/chunkify.rs

```diff
@@ -0,0 +1,172 @@
+#![cfg(feature = "chunkify")]
+
+// You strike me as a message that has never been chunkified
+// I'm sure I don't know what you mean, You forget yourself
+// There's a million bytes I haven't sent
+// but just you wait, just you wait
+//
+// Where are you taking me?
+// I'm about to change your life
+//
+// The conversation lasted two seconds, maybe three seconds
+// Everything we said in total agreement, it's
+// A dream and it's a bit of a dance
+// A bit of a posture, it's a bit of a stance
+// Each chunk gone by as the bytes advance,
+// Jetstream came through, with message enhanced!
+//
+// I will always be chunkified ...
+
+use crate::{
+    error::{RpcError, RpcResult},
+    provider_main::get_host_bridge,
+};
+use log::{debug, error};
+use nats::{
+    jetstream::JetStream,
+    object_store::{Config, ObjectStore},
+    JetStreamOptions,
+};
+use once_cell::sync::OnceCell;
+use std::{
+    collections::HashMap,
+    sync::{Arc, RwLock},
+};
+
+/// maximum size of a message payload before it will be chunked
+/// nats currently uses 128kb chunk size so this should be at least 128KB
+const CHUNK_THRESHOLD_BYTES: usize = 1024 * 200; // 200KB
```

Comment:
should probably set this threshold to the same threshold as the host. Technically it'll still work, but it would be better for consistency to have both sides decide to chunk on the same watermark.

Reply:
set to 700KB